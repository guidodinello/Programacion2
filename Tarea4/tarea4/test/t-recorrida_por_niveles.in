# Prueba de tiempo recorridaPorNiveles
tiempo_recorridaPorNiveles 500 3000 2
Fin
