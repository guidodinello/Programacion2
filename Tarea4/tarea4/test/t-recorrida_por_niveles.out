1>#  Prueba de tiempo recorridaPorNiveles.
2>
 Construyendo el árbol. 
 Imprimiendo el árbol. 
 Liberando el árbol. 
3>Fin.
